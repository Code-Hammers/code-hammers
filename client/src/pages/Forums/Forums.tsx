import React from "react";

const Forums = (): JSX.Element => {
  return (
    <div className="min-h-screen bg-gray-100 flex flex-col items-center justify-center">
      <h1 className="text-4xl font-extrabold mb-4">FORUMS</h1>
    </div>
  );
};

export default Forums;
