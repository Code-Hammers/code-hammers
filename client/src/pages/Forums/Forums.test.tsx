import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Forums from "./Forums";

describe("Forums Page", () => {
  it("renders the test H1 correctly", () => {
    const { getByText } = render(<Forums />);
    const title = getByText("FORUMS");
    expect(title).toBeInTheDocument();
  });

  it("matches the snapshot", () => {
    const { asFragment } = render(<Forums />);
    expect(asFragment()).toMatchSnapshot();
  });
});
