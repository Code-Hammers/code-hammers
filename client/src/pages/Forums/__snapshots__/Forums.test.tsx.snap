// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forums Page matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="min-h-screen bg-gray-100 flex flex-col items-center justify-center"
  >
    <h1
      class="text-4xl font-extrabold mb-4"
    >
      FORUMS
    </h1>
  </div>
</DocumentFragment>
`;
